subsys/esb/esb.c

```diff
@@ -781,7 +791,10 @@ static bool rx_fifo_push_rfbuf(uint8_t pipe, uint8_t pid)
 
 static void esb_timer_handler(nrf_timer_event_t event_type, void *context)
 {
-
+	nrf_timer_event_clear(esb_timer.p_reg, NRF_TIMER_EVENT_COMPARE1);
```

Comment:
`nrf_timer_int_enable_check` returns true but `nrf_timer_event_check`  returns false.
I assume it's because the shorts are configured for timer reset when compare event is triggered. I've made the changes and commented the code.